File: 200_LeetCode_Questions/Python/D02P3_LeetCode_1512.py
# Python3 , By:- Harsh Udai
# Que. Number of Good Pairs

class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        count=0
        for i in range(0,len(nums)):
            for j in range(i+1,len(nums)):
                if(i<j and nums[i]==nums[j]):
                    count+=1
                    
                    
        return count
        

File: 200_LeetCode_Questions/C++/D02P3_LeetCode_1512.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int count=0;
        for(int i=0;i<nums.size();i++){
            for(int j=0;j<nums.size();j++){
                if(i!=j && nums[i]==nums[j] && i<j)count++;
            }
        }
        return count;
        
    }
};